readme.md

```diff
@@ -202,6 +203,21 @@ got.stream('github.com')
 
 `redirect` event to get the response object of a redirect. The second argument is options for the next request to the redirect location.
 
+##### .on('uploadProgress', progress)
+##### .on('downloadProgress', progress)
```

Comment:
Should probably clarify that these can be used with the `Promise` interface too. It's a little ambiguous now since they're documented under `got.stream`.